### PR TITLE
Prow setup for knative/eventing-sources

### PR DESCRIPTION
**Pull request by n3wscott**
/lint

I also need someone with rights to run `make update-config`.

### CODE REVIEW

File: ci/prow/config.yaml [hunk 1]
```diff
@@ -683,6 +683,140 @@ presubmits:
...
+        - "--scenario=kubernetes_execute_bazel"
+        - "--clean"
+        - "--job=$(JOB_NAME)"
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://knative-prow/pr-logs"
+        - "--" # end bootstrap args, scenario args below
+        - "--" # end kubernetes_execute_bazel flags (consider following flags as text)
+        - "./test/presubmit-tests.sh"
```
Comment (adrcunha) [inline ci/prow/config.yaml]: a) You don't have the test scripts in the repo, all your Prow jobs will fail.
b) Please include the CI and coverage postsubmit jobs.
c) Please include the changes to testgrid config as well.

Comment (n3wscott) [reply ci/prow/config.yaml]: added the cron jobs.

Comment (n3wscott) [reply ci/prow/config.yaml]: added test grid changes. I can not run the makefile though

**Comment by n3wscott** [comment]
ok PTAL @adrcunha thanks!!

File: ci/prow/config.yaml [hunk 2]
```diff
@@ -1707,6 +1841,88 @@ periodics:
...
+      - "--clean"
+      - "--job=$(JOB_NAME)"
+      - "--repo=github.com/knative/eventing-sources"
+      - "--root=/go/src"
+      - "--service-account=/etc/service-account/service-account.json"
+      - "--upload=gs://knative-prow/logs"
+      - "--timeout=90" # 1.5h
+      - "--" # end bootstrap args, scenario args below
+      - "--" # end kubernetes_execute_bazel flags (consider following flags as text)
+      - "./hack/release.sh"
```
Comment (adrcunha) [inline ci/prow/config.yaml]: You're gonna overwrite (or get overwritten by) knative/eventing releases. I'm not sure that is intended...

Comment (n3wscott) [reply ci/prow/config.yaml]: oh no! yeah not intended. https://github.com/knative/eventing-sources/pull/9/

**Comment by n3wscott** [comment]
> you also missed adding eventing-sources repo to the tide section in the beginning of the config file.

Done.

**Pull request merged**